### PR TITLE
After proposal execution remove time counter (#2466)

### PR DESCRIPTION
**Pull request by chrlschwb**
issue #2466
After proposal is executed, page state changes from "Gracing" to "Executed".
Using this state variable we remove time counter after proposal execution.

### CODE REVIEW

**Comment by traumschule** [comment]
please merge https://github.com/chrlschwb/pioneer/pull/1 in here

**Comment by chrlschwb** [comment]
description: issue #2466
I have closed duplicate PR and left this one

**Comment by traumschule** [comment]
It could still profit from prettier.

**Comment by chrlschwb** [comment]
redeployed after updating the code from dev branch of main repo

File: packages/ui/src/app/pages/WorkingGroups/WorkingGroupsOpening.tsx [hunk 1]
```diff
@@ -149,7 +149,10 @@ export const WorkingGroupOpening = () => {
               <StatusBadge />
             </BadgesRow>
             <Statistics>
-              <DurationStatistics title="Time Left" value={opening.expectedEnding} />
+              {
+                OpeningStatuses.CANCELLED ? "" :
```
Comment (traumschule) [inline packages/ui/src/app/pages/WorkingGroups/WorkingGroupsOpening.tsx:153]: Do you try  only show when canceled?
Could be done in `DurationStatistics` instead with something like `if (!duraiton) return <></>`

**Review by traumschule — CHANGES_REQUESTED** [review]
please don't forget to enable prettier or use `yarn lint`

File: packages/ui/src/common/components/statistics/DurationStatistics.tsx [hunk 2]
```diff
@@ -29,7 +29,7 @@ export const DurationStatistics = (props: DurationStatisticsProps) => {
   const duration = Date.parse(props.value) - (props.from ? Date.parse(props.from) : Date.now())
   return (
     <StatisticItem {...props}>
-      <DurationValue value={formatDurationDate(duration)} />
+      {duration > 0 ? <DurationValue value={formatDurationDate(duration)} /> : <></>}
```
Comment (traumschule) [inline packages/ui/src/common/components/statistics/DurationStatistics.tsx:32]: Please resubmit as new PR with only this change:
```suggestion
      {duration && <DurationValue value={formatDurationDate(duration)} />}
```

**Comment by traumschule** [comment]
You shouldn't skip commit hooks:
```
Run yarn lint
@joystream/pioneer
 | 
 | /home/runner/work/pioneer/pioneer/packages/ui/src/app/pages/WorkingGroups/WorkingGroupsOpening.tsx
 |   82:23  error  React Hook "useMemo" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?  react-hooks/rules-of-hooks
 | 
 | /home/runner/work/pioneer/pioneer/packages/ui/src/council/hooks/useCouncilRemainingPeriod.ts
 |   2:30  warning  'concatMap' is defined but never used   @typescript-eslint/no-unused-vars
 |   2:[4](https://github.com/Joystream/pioneer/pull/3864/checks#step:5:5)1  warning  'EMPTY' is defined but never used       @typescript-eslint/no-unused-vars
 |   2:48  warning  'merge' is defined but never used       @typescript-eslint/no-unused-vars
 |   2:[5](https://github.com/Joystream/pioneer/pull/3864/checks#step:5:6)5  warning  'Observable' is defined but never used  @typescript-eslint/no-unused-vars
 |   2:67  warning  'of' is defined but never used          @typescript-eslint/no-unused-vars
 | 
 | /home/runner/work/pioneer/pioneer/packages/ui/src/forum/components/ForumPageHeader.tsx
 |   [6](https://github.com/Joystream/pioneer/pull/3864/checks#step:5:7):10  warning  'TextMedium' is defined but never used  @typescript-eslint/no-unused-vars
 | 
 | /home/runner/work/pioneer/pioneer/packages/ui/src/forum/components/category/LatestPost.tsx
 |    5:10  warning  'Loading' is defined but never used             @typescript-eslint/no-unused-vars
 |   1[7](https://github.com/Joystream/pioneer/pull/3864/checks#step:5:8):25  warning  'isLoading' is assigned a value but never used  @typescript-eslint/no-unused-vars
 | 
 | /home/runner/work/pioneer/pioneer/packages/ui/src/forum/components/category/PopularThread.tsx
 |    5:[10](https://github.com/Joystream/pioneer/pull/3864/checks#step:5:11)  warning  'Loading' is defined but never used             @typescript-eslint/no-unused-vars
 |   17:[11](https://github.com/Joystream/pioneer/pull/3864/checks#step:5:12)  warning  'isLoading' is assigned a value but never used  @typescript-eslint/no-unused-vars
 | 
 | /home/runner/work/pioneer/pioneer/packages/ui/src/memberships/components/MemberProfile/MemberRoleToggle.tsx
 |   35:21  warning  'currentDayEarnings' is assigned a value but never used  @typescript-eslint/no-unused-vars
 | 
 | ✖ [12](https://github.com/Joystream/pioneer/pull/3864/checks#step:5:13) problems (1 error, 11 warnings)
 | 
 | `yarn lint:fix` failed with exit code 1
```

**Comment by traumschule** [comment]
```
 | /home/runner/work/pioneer/pioneer/packages/ui/src/accounts/components/AccountItem/AccountItem.tsx
 |   15:10  warning  'BN_ZERO' is defined but never used  @typescript-eslint/no-unused-vars
 | 
 | /home/runner/work/pioneer/pioneer/packages/ui/src/app/pages/Settings/Settings.tsx
 |   10:10  warning  'LanguageSelect' is defined but never used  @typescript-eslint/no-unused-vars
 | 
 | /home/runner/work/pioneer/pioneer/packages/ui/src/app/pages/WorkingGroups/WorkingGroupsOpening.tsx
 |   [8](https://github.com/Joystream/pioneer/actions/runs/4019926042/jobs/6907317401#step:5:9)2:23  error  React Hook "useMemo" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?  react-hooks/rules-of-hooks
 | 
 | /home/runner/work/pioneer/pioneer/packages/ui/src/council/components/election/CandidateVote/CandidateVote.tsx
 |   10:1[9](https://github.com/Joystream/pioneer/actions/runs/4019926042/jobs/6907317401#step:5:10)  warning  'TooltipDefault' is defined but never used  @typescript-eslint/no-unused-vars
 | 
 | /home/runner/work/pioneer/pioneer/packages/ui/src/forum/types/ForumCategoryWithDetails.ts
 |   3:63  warning  'ForumModerator' is defined but never used  @typescript-eslint/no-unused-vars
```

**Comment by traumschule** [comment]
Suggest to recreate this as well: `Files changed 2,718`

**Pull request closed without merging**